src/client-agent/start_agent.c

```diff
@@ -426,3 +427,55 @@ static void send_msg_on_startup(void){
 
     send_msg(fmsg, -1);
 }
+
+/**
+ * @brief Send agent stopped message to server before exit
+ * */
+void send_agent_stopped_message() {
+    size_t msg_length;
+    ssize_t recv_b = 0;
+
+    char *tmp_msg;
+    char msg[OS_MAXSTR + 2] = { '\0' };
+    char buffer[OS_MAXSTR + 1] = { '\0' };
+    char cleartext[OS_MAXSTR + 1] = { '\0' };
+
+    snprintf(msg, OS_MAXSTR, "%s%s", CONTROL_HEADER, HC_SHUTDOWN);
+
+    /* Send start up message */
```

Comment:
Does should be "Send shut down message"?

Reply:
Fixed